dev-client/src/screens/LocationScreens/components/CreateSiteButton.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {useCallback} from 'react';
+import {useTranslation} from 'react-i18next';
+import {Button} from 'native-base';
+
+import {useNavigation} from 'terraso-mobile-client/navigation/hooks/useNavigation';
+import {Coords} from 'terraso-mobile-client/model/map/mapSlice';
+import {Icon} from 'terraso-mobile-client/components/icons/Icon';
+import {Box} from 'terraso-mobile-client/components/NativeBaseAdapters';
+
+type Props = {coords: Coords};
+
+export const CreateSiteButton = ({coords}: Props) => {
+  const {t} = useTranslation();
+  const navigation = useNavigation();
+
+  const onCreate = useCallback(() => {
+    navigation.navigate('CREATE_SITE', {coords});
+  }, [navigation, coords]);
+
+  return (
+    <Box paddingVertical={'20px'}>
```

Comment:
I did it this way because it maintained the spacing that existed before (which was `mt={5}` for this Box and for the element after it) and there was no string defined under `theme.space` that corresponded to 20px

Reply:
Do you want to add an `ml` to themes? Or as Courtney if we can use 16 or 24px here?

Reply:
I'll actually take the padding out of the button itself and let the parent figure out the padding. I'll put it at "md" (16px) for now, I'm guessing the designs for that screen aren't pixel perfect as currently it's 21px.